util/llpcInternal.cpp

```diff
@@ -115,7 +115,11 @@ CallInst* EmitCall(
         }
     }
 
-    return builder.CreateCall(pFunc, args);
+    auto pCall = builder.CreateCall(pFunc, args);
+    pCall->setCallingConv(CallingConv::C);
+    pCall->setAttributes(pFunc->getAttributes());
```

Comment:
Maybe you can provide more detail internally, but I don't understand how this can be necessary. The effective attributes of a function call should always be the union of the attributes of the function and those of the call, so it's surprising that having them on the call itself presumably makes a difference.